vite.config.js

```diff
@@ -65,6 +72,19 @@ export default defineConfig(({ mode }) => ({
 		}),
 		VitePWA(PWAConfig),
 	],
+	root: path.resolve(__dirname),
+	resolve: {
+		alias: {
+			"~bootstrap": path.resolve(__dirname, "node_modules/bootstrap"),
+		},
+	},
+	css: {
+		preprocessorOptions: {
+			scss: {
+				api: "modern-compiler",
+			},
+		},
+	},
```

Comment:
These are the same changes from Falak's old PR?

Reply:
Maybe, it was one of the things that cute talks about when using sass and I didn't see it in there when I was writing the resolver and the manual chunks so I did add it as well. I'd have to look at the old PR 🤔 this was branched from the messed up main so it may have been in a PR but didn't merge  🤷🏽‍♀️